### PR TITLE
HA: Data loss -> RPO

### PR DESCRIPTION
**Pull request by as51340**
### Description

In HA community a better term for data loss is recovery point objective. 

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

File: pages/clustering/high-availability.mdx [hunk 1]
```diff
@@ -416,7 +416,7 @@ are all disabled.
 
 <Callout type="warning">
 
-Under certain extreme scenarios, the current implementation of HA could lead to data loss. These are environments with high volume of transactions 
+Under certain extreme scenarios, the current implementation of HA could lead to having Recovery Point Objective (RPO) != 0. These are environments with high volume of transactions
```
Comment (kgolubic) [inline pages/clustering/high-availability.mdx:419]: I know that this is a standard way, `RPO!=0`, but I suggest replacing notation with words. Something along the lines of: 

`Under certain extreme scenarios, the current implementation of High Availability (HA) could result in a Recovery Point Objective (RPO) greater than zero, meaning some data loss could occur.`

**Review by kgolubic — CHANGES_REQUESTED** [review]
Please update the sentence to be more user friendly.

**Comment by as51340** [comment]
> Please update the sentence to be more user friendly.

Any advices?

**Comment by kgolubic** [comment]
> > Please update the sentence to be more user friendly.
> 
> Any advices?

`Under certain extreme scenarios, the current implementation of High Availability (HA) could result in a Recovery Point Objective (RPO) greater than zero, meaning some data loss could occur.`

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**